code/script.js

```diff
@@ -1,9 +1,29 @@
 // A function that adds and remove the class "active" on the section you click on.
 // We haven't really talked about ´this´ yet, but we will... ;)
 // console.log(this) to see how it works
+
+/*
 function toggle() {
   this.classList.toggle("active")
 }
 
 // Selects an HTML element, and calls a function which will be executed when the element is clicked.
-document.getElementById("section1").onclick = toggle
+document.getElementById("question1").onclick = toggle
+
+*/
+
+console.log(this)
```

Comment:
We know it works, so you can delete all `console.log()` before making a pull request! It's more for you to try your code so it works!
```suggestion
```